Error parsing `desktop.html` and `mobile.html`

Still during step 4:

```
.build/venv/bin/pot-create --config lingua-client.cfg --output nyon/locale/nyon-client.pot nyon/templates/edit.html nyon/templates/desktop.html nyon/templates/routing.html nyon/templates/api/apihelp_old.html nyon/templates/api/xapihelp.html nyon/templates/api/apihelp.html nyon/templates/mobile.html nyon/static-ngeo/js/nyonmodule.js nyon/static-ngeo/js/mobile.js    .build/config.yaml development.ini
Error parsing ./nyon/templates/desktop.html at line 3 column 102
Error parsing ./nyon/templates/mobile.html at line 6 column 74
Traceback (most recent call last):
  File ".build/venv/bin/pot-create", line 11, in <module>
    sys.exit(main())
  File "/.build/venv/local/lib/python2.7/site-packages/lingua/extract.py", line 327, in main
    for message in extractor(real_filename, options):
  File "/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/lib/lingua_extractor.py", line 124, in __call__
    self.env = bootstrap(filename)
  File "/.build/venv/local/lib/python2.7/site-packages/pyramid/paster.py", line 131, in bootstrap
    app = get_app(config_uri, options=options)
  File "/.build/venv/local/lib/python2.7/site-packages/pyramid/paster.py", line 31, in get_app
    global_conf=options)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 271, in loadobj
    global_conf=global_conf)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 296, in loadcontext
    global_conf=global_conf)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 320, in _loadconfig
    return loader.get_context(object_type, name, global_conf)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 450, in get_context
    global_additions=global_additions)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 559, in _pipeline_app_context
    APP, pipeline[-1], global_conf)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 454, in get_context
    section)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 476, in _context_from_use
    object_type, name=use, global_conf=global_conf)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 406, in get_context
    global_conf=global_conf)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 296, in loadcontext
    global_conf=global_conf)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 328, in _loadegg
    return loader.get_context(object_type, name, global_conf)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 620, in get_context
    object_type, name=name)
  File "/.build/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 640, in find_egg_entry_point
    pkg_resources.require(self.spec)
  File "/.build/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 968, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/.build/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 859, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (PyYAML 3.12 (/.build/venv/lib/python2.7/site-packages), Requirement.parse('PyYAML==3.11'), set(['pykwalify']))
CONST_Makefile:686: recipe for target 'nyon/locale/nyon-client.pot' failed
make[1]: *** [nyon/locale/nyon-client.pot] Error 1
make[1]: Leaving directory ''
Traceback (most recent call last):
  File ".build/venv/bin/c2ctool", line 11, in <module>
    sys.exit(main())
  File "/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 98, in main
    c2ctool.upgrade()
  File "/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 222, in upgrade
    self.step4()
  File "/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 436, in step4
    check_call(["make", "-f", self.options.file, "build"])
  File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make', '-f', 'instance_v2.mk', 'build']' returned non-zero exit status 2
CONST_Makefile:1237: recipe for target 'upgrade4' failed
make: *** [upgrade4] Error 1
```

It's strange because the referenced parsing error does not exist in my files...

--- a/c2cgeoportal/scaffolds/update/CONST_versions_requirements.txt
+++ b/c2cgeoportal/scaffolds/update/CONST_versions_requirements.txt
@@ -25,6 +25,7 @@ PasteScript==2.0.2
 polib==1.0.7
 postmarkup==1.2.2
 psycopg2==2.6.2
+pykwalify==1.5.1
 pyramid-chameleon==0.3
 pyramid-debugtoolbar==3.0.4
 pyramid-mako==1.0.2
@@ -32,7 +33,6 @@ pyramid-multiauth==0.8.0
 pyramid-tm==1.0
 python-dateutil==2.5.3
 pytz==2016.7
-PyYAML==3.12
 repoze.lru==0.6
 requests==2.11.1
 simplejson==3.8.2

